xonsh/built_ins.py

```diff
@@ -720,6 +724,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_execer__ = execer
     builtins.__xonsh_commands_cache__ = CommandsCache()
     builtins.__xonsh_all_jobs__ = {}
+    builtins.__xonsh_enabled_builtin_commands__ = set(all_builtin_commands.keys())
```

Comment:
Yeah, I guess that I am not totally sold on the notion of builtin commands distinct from aliases. Maybe that means that aliases should be renamed :).  Basically my objections are that (1) it takes away from some of the agency of the users since it seems like they shouldn't touch builtins, and (2) it becomes another thing for folks to learn, which makes it harder to teach. I, of course, could be wrong about this :)

I guess the problem that you are trying to solve with this is that you want users to be able to load/unload these commands en masse?  Is that your main concern?